masschallenge/django-accelerator
# AC-6882 :: Sitetree navigation is the standard for all users

**Changes introduced in [AC-6882](https://masschallenge.atlassian.net/browse/AC-6882)**

- remove use_site_tree_side_nav from the program family 
- remove use_site_tree_side_nav flag

**How to test:**
 - As an admin got to any [program family](http://localhost:8181/admin/mc/programfamily/22/change/) and see that the use_site_tree_side_nav check box is not present.

 - Masquerade as any entrepreneur and give them any finalist role grant and see that they have the site_tree_side_nav
 - Masquerade as any expert and give them any mentor or judging role grant and see that they have the site_tree_side_nav
 On development an expert or entrepreneur with a role grant of a program that doesn't have the site_tree_side_nav enabled sees the old `menu_renderer`.
